tables/ka-ge-g1.utb

```diff
@@ -0,0 +1,215 @@
+# liblouis: Kartvelian, Georgia grade 1 table
```

Comment:
This table will need some metadata. @jkinslayer How about the following?

```
#+locale: ccs
#+type: literary
#+contraction: no
```

"ccs" is the official ISO 639-5 tag for the family of Kartvelian (South Caucasian) languages. Is this table really specific to Georgia?

Do you also have documentation about this braille code, like you have for [Romanian](https://www.pharmabraille.com/braille-codes/romania-braille-code)?